test/examples/07_typedef/typedef.cpp

```diff
@@ -0,0 +1,8 @@
+#include "typedef.h"
+
+namespace chimera_test
+{
+
+//==============================================================================
+
+} // namespace chimera_test
```

Comment:
Do we even need this file?  Everything should work without it right?

Reply:
We don't. It seems I add source files habitually even it's empty. Let me remove this.